src/components/ModiteList/formatModites.js

```diff
@@ -11,33 +12,38 @@ onmessage = function(event) {
     return '☀️';
   };
 
+  const isProject = modites.length && modites[0].recordType === 'project';
+
   const filtered = modites
     .filter(
-      modite =>
-        modite.real_name.toLowerCase().indexOf(filter.toLowerCase()) > -1
+      modite => {
+        const name = isProject ? 'name' : 'real_name';
+        return modite[name].toLowerCase().indexOf(filter.toLowerCase()) > -1;
+      }
     )
     .sort((prev, next) => {
-      const prevName = prev.profile.last_name;
-      const nextName = next.profile.last_name;
+      const prevName = isProject ? prev.name : prev.profile.last_name;
+      const nextName = isProject ? next.name : next.profile.last_name;
 
       if (prevName < nextName) return -1;
       if (prevName > nextName) return 1;
       return 0;
     })
     .map(modite => ({
       ...modite,
-      localTime: date.toLocaleString(locale, {
+      'real_name': modite.real_name || modite.name,
```

Comment:
Quotes aren't needed here. If it was `real-name` then it'd be required but underscore is fine.

Reply:
true.  fixed